Reword replicated stateful app task (#33791)
kubernetes/website: content/en/docs/tasks/run-application/run-replicated-stateful-application.md

mysql-2   2/2       Running   0          15m       10.244.5.27   kubernetes-node-9l2t
```

Then, drain the Node by running the following command, which cordons it so
no new Pods may schedule there, and then evicts any existing Pods.
Replace `<node-name>` with the name of the Node you found in the last step.

{{< caution >}}

> Draining a Node can impact other workloads and applications that are
> running on the same node, so only do the following step in a test
> cluster.

Reviewer: Slight edits for clarity
```suggestion
Draining a Node can impact other workloads and applications 
running on the same node. Only perform the following step in a test
cluster.
```
